src/Sulu/Bundle/PreviewBundle/Resources/js/index.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import {bundleReady} from 'sulu-admin-bundle/services';
+import initializer from 'sulu-admin-bundle/services/Initializer';
+import {sidebarViewRegistry} from 'sulu-admin-bundle/containers/Sidebar';
+import Preview, {previewConfigStore} from './views/Preview';
+
+initializer.addUpdateConfigHook('sulu_preview', (config: Object) => {
+    previewConfigStore.setConfig(config);
```

Comment:
I am not totally sure, but I think until now a `store` is something communicating with the server, i.e. sending some requests. I think this is not the case for this class, as it gets the config passed. We have the concept of registries, which contain instances of some other classes, but that also does not match here... Maybe just call it `previewConfig`? Also to be discussed.

Reply:
@chirimoya

Reply:
`previewStore` only